src/main/java/com/gradle/CiUtils.java

```diff
@@ -0,0 +1,74 @@
+package com.gradle;
+
+import org.gradle.api.provider.ProviderFactory;
+
+final class CiUtils {
```

Comment:
What are your thoughts on naming this just `CI` or `CIs`? I personally dislike the `Utils` naming convention as it ends up making it such that every class is named `XXXUtils`.

Reply:
No strong opinion, I followed the existing pattern 😅 
I find `CiUtils` more meaningful than `CI / CIs` though.

I usually only use the `Utils / Helper` suffix for those classes exposing static methods.